### PR TITLE
USH-4826: Fix alignment of button

### PR DESCRIPTION
**Pull request by MartinRiese**
## Product Description

https://github.com/dimagi/commcare-hq/commit/f9e6bcae2373a047cc7eb8b75b9dbbb03968a2a4 made changes that broke the appearance attribute combination `button-select text-align-right`. Revert some of the changes in a simplified manner.

Specifically the css classes `text-center` and `text-end` where not set anymore on the question if it was of type button. Furthermore, the width of the column was not reduced, the case of a missing label causing the cell to overflow

## Technical Summary

https://dimagi.atlassian.net/browse/USH-4826

## Feature Flag

No

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js [hunk 1]
```diff
@@ -1030,13 +1030,8 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
             self.labelWidth = "";
             self.questionTileWidth = `col-md-${columnWidth}`;
         } else {
-            if (self.isLabel || self.isButton) {
-                self.controlWidth = "";
```
Comment (Jtang-1) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js:1034]: why was this changed needed?

**Review by Jtang-1 — APPROVED** [review]

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js (hunk 1 above)
Comment (MartinRiese) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js:1034]: The bug had the case with isButton = true and hasLabel = false (see line1037). Before this change only `offset-md-4` was applied but the width was still 12 causing the column to over flow on the right. After it is `col-md-8 offset-md-4`, moving the column to the right but also shrinking it by 4.
